hydra-node/src/Hydra/Node/Run.hs

```diff
@@ -91,7 +95,7 @@ run opts = do
           withAPIServer apiHost apiPort party apiPersistence (contramap APIServer tracer) chain pparams (wireClientInput wetHydraNode) $ \server -> do
             -- Network
             let networkConfiguration = NetworkConfiguration{persistenceDir, signingKey, otherParties, host, port, peers, nodeId}
-            withNetwork tracer networkConfiguration (wireNetworkInput wetHydraNode) $ \network -> do
+            withNetwork tracer currentHydraVersionedProtocol networkConfiguration (wireNetworkInput wetHydraNode) $ \network -> do
```

Comment:
Should: not need to know which network layer we actually use here

Taking parameter and needing to import `Hydra.Network.Ouroboros` here closely ties the code in `Hydra.Node.Network` and this site together.